app/src/main/java/woowacourse/paint/model/DrawingHistory.kt

```diff
@@ -0,0 +1,22 @@
+package woowacourse.paint.model
+
+import android.graphics.Paint
+import android.graphics.Path
+
+class DrawingHistory {
+
+    private val _history = mutableListOf<Drawing>()
+    val history: List<Drawing>
+        get() = _history.map {
+            it.copy(
+                path = Path(it.path),
+                paint = Paint(it.paint),
+            )
+        }.toList()
```

Comment:
아래 방식과 어떤 차이가 있나요?
아래 방식으로도 의도한대로? 동작할 거 같아요!
```suggestion
get() = _history.toList()
```

Reply:
아래 방식은 깊은 복사가 되지 않은 상태입니다! 
저도 [베리의 리뷰](https://github.com/woowacourse/android-paint/pull/8#discussion_r1346738747)를 훔쳐보다가 알게 되었습니다ㅋㅋㅋ